src/app/pages/errata-summary/hero/hero.js

```diff
@@ -34,29 +29,80 @@ function useBookInfo(book) {
     return info;
 }
 
-function useErrataHoverHtml(slug) {
-    const [html, setHtml] = useState('<p>...loading...</p>');
+const middle = '-135';
+
+function shiftIntoView(ref, leftOffset, setLeftOffset) {
+    const {left, right} = ref.current.getBoundingClientRect();
+    const {right: pageRight} = ref.current.closest('.page').getBoundingClientRect();
```

Comment:
what's your browser support like?  will it have `closest` ?  ie 11 doesn't have that

Reply:
We don't support IE, only Edge. Good point, though.